core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorAccessControl.java

```diff
@@ -659,6 +658,6 @@ default Optional<ViewExpression> getColumnMask(ConnectorSecurityContext context,
      */
     default List<ViewExpression> getColumnMasks(ConnectorSecurityContext context, SchemaTableName tableName, String columnName, Type type)
     {
-        return emptyList();
```

Comment:
Can we simply remove deprecated methods? They were deprecated already long enough.

Reply:
The policy was that we'll keep them for one more release, right? According to that yes, we can remove them. Though with the current release pace that's not that long :)

Reply:
#12998

Reply:
Thanks for removing the deprecated methods @ksobolew. Thanks @kokosing will close the PR.